### PR TITLE
Create pip package and deploy

### PR DESCRIPTION
**Pull request by just-paja**

### CODE REVIEW

File: bin/run.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+app="improtresk-api"
+
+DIR_ROOT="$(dirname "$(readlink -f "$0")")"
+
+cd $DIR_ROOT
+
+source ../../bin/activate
+
+python ../manage.py migrate
```
Comment (PetrDlouhy) [inline bin/run.sh]: Automatically triggering migrations could be dangerous. We need to be extra careful with this.

I would backup the database just prior to executing migrations (if the state of the database at that point is not backed up by something different). That could save our ass sometime.

Testing the migration of production database on development environment is also good to do prior migrating the production.

File: requirements.txt [hunk 2]
```diff
@@ -34,3 +34,4 @@ six==1.10.0
 virtualenv==15.1.0
 watchdog==0.8.3
 wrapt==1.10.8
+mysqlclient==1.3.9
```
Comment (just-paja) [inline requirements.txt]: We don't really need multiple database servers, no. Also, the risk is very little for database of our size.

File: improtreskApi/settings.py [hunk 3]
```diff
@@ -29,11 +24,12 @@
     AWS_SECRET_ACCESS_KEY = os.environ.get('DJANGO_AWS_SECRET_ACCESS_KEY')
     AWS_STORAGE_BUCKET_NAME = os.environ.get('DJANGO_AWS_STORAGE_BUCKET_NAME')
 
-# SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = True
+DEBUG = os.environ.get('DJANGO_DEBUG', False)
```
Comment (just-paja) [inline improtreskApi/settings.py]: `DJANGO_DEBUG=True ./manage.py runserver`

File: bin/run.sh (hunk 1 above)
Comment (just-paja) [inline bin/run.sh]: Well, database from production will be cloned into staging database. Every build from master is deployed to staging, so I think it's safe to assume that all migrations will be tested against -the- database.

Created #79 for database backup. We don't need it for initial release. We can also add automatic rollback when the deploy fails. Not a big problem.

File: bin/run.sh [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+python ../manage.py collectstatic --no-input
+python ../manage.py check --deploy
+python ../manage.py runserver 0.0.0.0:${DJANGO_PORT}
```
Comment (just-paja) [inline bin/run.sh]: @PetrDlouhy, Yes, we need to figure out how to run it.

File: bin/run.sh (hunk 1 above)
Comment (PetrDlouhy) [inline bin/run.sh]: It is not problem when the migration fails (at least on PostgreSQL), but when it corrupts data in unexpected way, that couldn't be clearly visible at the point of running that migration.

But I assume, that your solutions are for the project of our size OK.

File: improtreskApi/settings.py (hunk 3 above)
Comment (PetrDlouhy) [inline improtreskApi/settings.py]: OK, good enough solution, if it is only variable, that we will need to set. But please add this command at least to README.

File: requirements.txt (hunk 2 above)
Comment (PetrDlouhy) [inline requirements.txt]: OK

File: bin/run.sh (hunk 4 above)
Comment (PetrDlouhy) [inline bin/run.sh]: From [documentation](https://docs.djangoproject.com/en/dev/ref/django-admin/#runserver):

```
DO NOT USE THIS SERVER IN A PRODUCTION SETTING. It has not gone through security 
audits or performance tests. (And that’s how it’s gonna stay. We’re in the business of making 
Web frameworks, not Web servers, so improving this server to be able to handle a production 
environment is outside the scope of Django.)
```

**Pull request merged**